### PR TITLE
Update tests to remove some in favor of automated testing

### PR DESCRIPTION
**Pull request by geriux**
This PR updates some manual tests in favor of automated testing.

- Gallery block - Add Gallery Caption - [TC003] -> https://github.com/WordPress/gutenberg/pull/38571
- Gallery block - Add Gallery Image Caption - [TC04] -> https://github.com/WordPress/gutenberg/pull/38571
- Columns - Multiple slider in columns parent - [TC011] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548
- Columns - Check if Column placeholders is visible for the unselected state - [TC001] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548
- Columns - Appearance in vertical device position (vertical layout) - [TC004] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548
- Columns - Max limit of Column number - [TC002] -> https://github.com/WordPress/gutenberg/pull/51095
- Columns - Deep nesting is possible (iOS only)- [TC003] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5815
- Columns - Appearance in horizontal device position (horizontal layout) - [TC005] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548
- Columns - Mover direction - [TC006] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548
- Columns - Check if Column placeholder is render in nested structure - [TC007] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548
- Columns - Unselected Column placeholder is stretched to content in nested structure- [TC008] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548
- Spacer block - Spacer in horizontal layout works as expected - [TC001] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5501
- Buttons block - Render custom text color - [TC013] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5501
- Buttons block - Render custom background color - [TC012] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5501
- Buttons block - Render gradient background color - [TC014] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5501
- Buttons block - Check if selection / caret color matches font color - [TC016] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5501
- Buttons block - Edit text styles - [TC005] -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/5501

### CODE REVIEW

File: test-cases/gutenberg/columns.md
Comment (dcalhoun) [inline test-cases/gutenberg/columns.md:1]: I believe TC011 can be removed as well, if desired. https://github.com/wordpress-mobile/gutenberg-mobile/pull/5548

**Review by dcalhoun — APPROVED** [review]
🔥🔥🔥

Thank you! If you have time, please update the tracking spreadsheet with the latest status for these tests.

**Comment by geriux** [comment]
> 🔥🔥🔥
> 
> Thank you! If you have time, please update the tracking spreadsheet with the latest status for these tests.

Will do! Thank you for reviewing! 🚀

File: test-cases/gutenberg/columns.md
Comment (geriux) [reply test-cases/gutenberg/columns.md:1]: Nice catch! I'll remove it 👍

Comment (geriux) [reply test-cases/gutenberg/columns.md:1]: Updated in https://github.com/wordpress-mobile/test-cases/pull/142/commits/4d4fbb940e6b609ccdafc8b5641e233d029bb9b1

**Pull request merged**